config/secrets.yml

```diff
@@ -1,8 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
 development:
-  secret_key_base: 4c987fc2f10296e546c25ee1965d93e6995ed248c679e4e6153c6fa3ab5bb43fd2709322e535dbb911ac2d33d9daf79e0ad0fbd1c33ea6444a9ad201c7a73fb8
+  secret_key_base: be2c4a5ec154182c27c00a3ecc760bc552d768e34437574efd67e80415fda978863899994c9d1357bcdd6ba5ed8f8c42c89cbb87ceee64f1c452d8b540b9cd82
```

Comment:
Just curious, was this a purposeful rotation? Should we be rotating these periodically on other apps?

Reply:
Nope, for some reason the `rails app:upgrade` task replaced them, and it also doesn't matter in the case where the app has been down for 11 months, so I just shrugged.